tests/openssh.rs

```diff
@@ -822,3 +822,30 @@ async fn local_socket_forward() {
         assert!(output.status.success());
     }
 }
+
+#[tokio::test]
+#[cfg_attr(not(ci), ignore)]
+async fn test_sftp_subsystem() {
```

Comment:
Should we try to write a file to `/tmp` or something too just to check that the whole machinery actually works?

Reply:
Done!

I added writing and reading to `/tmp/file` to test the integration with `openssh-sftp-client` is working!